src/opnsense/scripts/interfaces/ifinfo.php

```diff
@@ -0,0 +1,110 @@
+#!/usr/local/bin/php
```

Comment:
Would be nicer to fold this into pluginctl, very similar functionality already exists there.

Reply:
sure :)